server.go

```diff
@@ -19,6 +20,32 @@ func fallback(w http.ResponseWriter, r *http.Request, reason string) {
 	http.Redirect(w, r, location, 302)
 }
 
+func getRedirect(txt []string, url string) (*Redirect, error) {
+	catch_alls := make(map[string]*Config)
```

Comment:
Re camel case: Done.

Re map vs array/slice: Done. I originally had the map because it was holding both the DNS record and the already parsed redirect type. The DNS record was used for logging. I'll update since this is no longer required. However, since the DNS records do not currently support any sort of order the preservation of order given by an array isn't really of benefit. All it's really doing is preserving the order of the records as returned by the DNS lookup which is inherently non-ordered.